### ISSUE TITLE
Shorten title of package

### ISSUE THREAD

**Issue by geanders** [rickzhouwen/HDtest#1]
The package title is much too long-- on some systems, I think a lot of the title will be cut off. Can we cut it down to 65 or less characters?

**Comment by rickzhouwen** [comment]
Certainly agreed!! How about "High dimensional test for means, covariances and white noises"? 62 characters?

**Comment by geanders** [comment]
Great! I'll make that change. 

We can also make the Description longer (closer to a paragraph). I usually also include grant support in that.

**Comment by rickzhouwen** [comment]
Sounds great! I will add something more (I will try with Rstudio first:). I guess I may need to use push or pull somehow

**Comment by geanders** [comment]
Yes, you will need to pull from the repository (use the down arrow in GitHub). I created a new branch called `devel` with all my new changes, so I wouldn't mess up the `master` branch until we're done.

**Comment by rickzhouwen** [comment]
Thanks a lot! I will learn more about it this weekend:) 

Let's keep updating this. Youngseok will share you his package/R codes (not quite a package) on testing spatial structures of MRF later too.
